packages/design-system-react/src/components/Button/Button.test.tsx

```diff
@@ -236,6 +236,142 @@ describe('Button', () => {
+          'focus-visible:outline',
+          'focus-visible:outline-2',
+          'focus-visible:outline-primary-default',
+          'focus-visible:outline-offset-2',
+        );
+      });
+
+      it('applies correct focus outline for inverse secondary button', () => {
+        render(
+          <Button variant={ButtonVariant.Secondary} isInverse>
+            Secondary Inverse Button
+          </Button>,
+        );
+        const button = screen.getByRole('button');
+        expect(button).toHaveClass(
+          'focus-visible:ring-0',
+          'focus-visible:outline',
+          'focus-visible:outline-2',
+          'focus-visible:outline-background-default',
+          'focus-visible:outline-offset-4',
+        );
+      });
+
+      it('applies correct focus outline for danger secondary button', () => {
+        render(
+          <Button variant={ButtonVariant.Secondary} isDanger>
+            Secondary Danger Button
+          </Button>,
+        );
+        const button = screen.getByRole('button');
+        expect(button).toHaveClass(
+          'focus-visible:ring-0',
+          'focus-visible:outline',
+          'focus-visible:outline-2',
+          'focus-visible:outline-primary-default',
+          'focus-visible:outline-offset-2',
+        );
+      });
+    });
+
+    describe('Tertiary Button Focus', () => {
+      it('applies correct focus outline for non-inverse tertiary button', () => {
+        render(
+          <Button variant={ButtonVariant.Tertiary}>Tertiary Button</Button>,
+        );
+        const button = screen.getByRole('button');
+        expect(button).toHaveClass(
+          'focus-visible:ring-0',
+          'focus-visible:outline',
+          'focus-visible:outline-2',
+          'focus-visible:outline-primary-default',
+          'focus-visible:outline-offset-2',
+        );
+      });
+
+      it('applies correct focus outline for inverse tertiary button', () => {
+        render(
+          <Button variant={ButtonVariant.Tertiary} isInverse>
+            Tertiary Inverse Button
+          </Button>,
+        );
+        const button = screen.getByRole('button');
+        expect(button).toHaveClass(
+          'focus-visible:ring-0',
+          'focus-visible:outline',
+          'focus-visible:outline-2',
+          'focus-visible:outline-background-default',
+          'focus-visible:outline-offset-4',
+        );
+      });
+
+      it('applies correct focus outline for danger tertiary button', () => {
+        render(
+          <Button variant={ButtonVariant.Tertiary} isDanger>
+            Tertiary Danger Button
+          </Button>,
+        );
+        const button = screen.getByRole('button');
+        expect(button).toHaveClass(
+          'focus-visible:ring-0',
+          'focus-visible:outline',
+          'focus-visible:outline-2',
+          'focus-visible:outline-primary-default',
+          'focus-visible:outline-offset-2',
+        );
+      });
+    });
+  });
```

Comment:
non-blocking: I think these focus states look great but I'm wondering if there is a way we can set it globally so there is consistency across all components 🤔 


https://github.com/user-attachments/assets/c6611da7-e09f-427b-a757-bb6a7d6058c1